app-config.yaml

```diff
@@ -293,6 +293,14 @@ dynatrace:
 # Plugin: Permission
 permission:
   enabled: ${PERMISSION_ENABLED}
+  rbac:
+    admin:
+      users:
+        - name: user:default/alice
```

Comment:
Let's remove this line.  We can keep the admin group but remove the alice user line

Reply:
Done